web/src/containers/ApdStateKeyPerson.js

```diff
@@ -0,0 +1,248 @@
+              </div>
+            ))}
+            <div className="ds-l-col--auto">
+              <FormLabel>Total</FormLabel>
+              <Dollars>{personTotalCost(person)}</Dollars>
+            </div>
+          </div>
+        }
+      >
+        Yes
+      </Choice>
+      <Button
+        variation="primary"
+        className="ds-u-margin-top--4"
+        onClick={collapse}
+      >
+        Done
+      </Button>
+    </div>
+  );
+};
+PersonForm.propTypes = {
+  collapse: PropTypes.func.isRequired,
+  number: PropTypes.number.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  handleYearChange: PropTypes.func.isRequired,
+  person: PropTypes.object.isRequired,
+  primary: PropTypes.bool.isRequired,
+  years: PropTypes.array.isRequired
+};
+
+class ApdStateKeyPerson extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { expanded: props.initialExpanded };
+  }
+
+  toggleExpanded = () => {
+    this.setState(prev => ({ expanded: !prev.expanded }));
+  };
+
+  render() {
+    const {
+      handleChange,
+      handleYearChange,
+      number,
+      person,
+      primary,
+      remove,
+      years
+    } = this.props;
+    const { expanded } = this.state;
+
+    return (
+      <Fragment>
+        <div className="visibility--screen">
+          {expanded ? (
+            <PersonForm
+              collapse={this.toggleExpanded}
+              handleChange={handleChange}
+              handleYearChange={handleYearChange}
+              number={number}
+              person={person}
+              primary={primary}
+              years={years}
+            />
+          ) : (
+            <Review
+              heading={`${number}. ${person.name}`}
+              editHref="#"
+              onEditClick={this.toggleExpanded}
+              editContent={
+                <div>
+                  <Button
+                    size="small"
+                    variation="transparent"
+                    onClick={this.toggleExpanded}
+                  >
+                    Edit
+                  </Button>
+                  {!primary && (
+                    <Fragment>
+                      |
+                      <Button
+                        size="small"
+                        variation="transparent"
+                        onClick={remove}
+                      >
+                        Remove
+                      </Button>
+                    </Fragment>
+                  )}
+                </div>
+              }
+            >
+              {person.position}
+              <br />
+              Total cost:{' '}
+              <Dollars>{person.hasCosts ? personTotalCost(person) : 0}</Dollars>
+              {primary ? (
+                <Fragment>
+                  <br />
+                  Primary APD Point of Contact
+                </Fragment>
+              ) : null}
+            </Review>
+          )}
+        </div>
+        <div className="visibility--print">
```

Comment:
I don't recommend this as a pattern going forward, but it feels like a pretty powerful demo to me of how useful it'd be to have a designed-for-print page with no interactions, no math, no logic (or very little, anyway), etc.  Just display the danged thing, and end up with fewer pages!

![printers gone wild](https://media.giphy.com/media/yUrUb9fYz6x7a/giphy.gif)